### DISCUSSION TITLE
Error while calling restoreNVM: The NVM file ID must be set before serializing

### DISCUSSION THREAD

**Discussion by renatohtpc** [zwave-js/zwave-js-ui#3917] [General]
I am running zwave-js-ui inside a docker container. I am trying to migrate from a Zooz 700 controller to a Zooz 800 one.

Steps I followed.

Took an NVM backup of the 700 controller
Stopped the docker container
Removed the 700 controller
Modified the docker-compose.yml entry for /dev/serial/by-id/ to point to the new controller
Restarted the docker container.

The system came up only showing the controller (800) as the only entry.
Tried restoring the NVM back, but ended up with the above error message.

Any suggestions?

Thanks
Renato

**Comment by robertsLando** [comment]
cc @AlCalzone

**Reply by AlCalzone** [reply]
Likely a duplicate of https://github.com/zwave-js/node-zwave-js/discussions/7199
